code/cgame/cg_players.c

```diff
@@ -726,6 +731,15 @@ static const char *CG_GetTeamColors( const char *color, team_t team ) {
 		default: break;
     }
 
+	if ( !strcmp(color,"cid") ) {
+		if (clientNum < 7) {
+			return va("00%i",clientNum);
+		}
+		else {
+			return va("0%i",CG_ToBase7(clientNum));
+		}
+	}
```

Comment:
Could you do the following?
```c
		if ( clientNum < 7 ) {
			return va( "00%i", clientNum );
		}
		return va( "0%i", CG_ToBase7( clientNum ) );
```
This `else` condition is unnecessary. Also note about whitespaces I put on that, it should bbe like John Carmack or some id dev coding style.

Reply:
done